packages/block-library/src/block/edit-title.native.js

```diff
@@ -6,29 +6,33 @@ import { Text, View } from 'react-native';
 /**
  * WordPress dependencies
  */
-import { Icon } from '@wordpress/components';
+import { Icon, GlobalStylesContext } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { withPreferredColorScheme } from '@wordpress/compose';
 import { help, lock } from '@wordpress/icons';
+import { useContext } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import styles from './editor.scss';
 
 function EditTitle( { getStylesFromColorScheme, title } ) {
-	const lockIconStyle = getStylesFromColorScheme(
-		styles.lockIcon,
-		styles.lockIconDark
-	);
-	const titleStyle = getStylesFromColorScheme(
-		styles.title,
-		styles.titleDark
-	);
-	const infoIconStyle = getStylesFromColorScheme(
-		styles.infoIcon,
-		styles.infoIconDark
-	);
+	const globalStyles = useContext( GlobalStylesContext );
```

Comment:
Nitpick: I noticed we already have a function to get global styles.

https://github.com/WordPress/gutenberg/blob/c605917977350b46c7852dbdf0fd722f4c1bfe24/packages/components/src/mobile/global-styles-context/index.native.js#L89-L93

Reply:
Thanks for spotting that! Updated in https://github.com/WordPress/gutenberg/pull/55399/commits/7b1e40b62c7669539c5e40f5d9f86856d530ac8b.